### PR TITLE
Avoid unneeded calls to pow() and log().

### PR DESCRIPTION
**Pull request by atgeirr**
Gives a minor speedup to this code (10+ % faster assembly).

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

File: opm/material/binarycoefficients/Brine_CO2.hpp [hunk 1]
```diff
@@ -206,27 +206,28 @@ class Brine_CO2 {
         Evaluation a_mix = aMix_(temperature, yH2O, highTemp);
         Scalar b_CO2 = bCO2_(highTemp); 
         Evaluation b_mix = bMix_(yH2O, highTemp);
+        Evaluation Rt15 = R * pow(temperature, 1.5);
```
Comment (bska) [inline opm/material/binarycoefficients/Brine_CO2.hpp:209]: Could this be `const`?

File: opm/material/binarycoefficients/Brine_CO2.hpp [hunk 2]
```diff
@@ -206,27 +206,28 @@ class Brine_CO2 {
...
 
         Evaluation lnPhiCO2;
         if (spycherPruess2005) {
+            Evaluation logVpb_V = log((V + b_CO2) / V);
```
Comment (bska) [inline opm/material/binarycoefficients/Brine_CO2.hpp:213]: Could this be `const`?

File: opm/material/binarycoefficients/Brine_CO2.hpp [hunk 3]
```diff
@@ -262,22 +263,24 @@ class Brine_CO2 {
         Evaluation a_mix = aMix_(temperature, yH2O, highTemp);
         Scalar b_H2O = bH2O_(highTemp); 
         Evaluation b_mix = bMix_(yH2O, highTemp);
+        Evaluation Rt15 = R * pow(temperature, 1.5);
+        Evaluation logVpb_V = log((V + b_mix) / V);
```
Comment (bska) [inline opm/material/binarycoefficients/Brine_CO2.hpp:267]: Could these be `const`?

**Review by bska — COMMENTED** [review]
I appreciate it.  If some of the new cached results could be `const` then that would be even better.

**Comment by atgeirr** [comment]
Const-ified both changed functions.

**Review by bska — APPROVED** [review]
Greatly appreciated.  I'll merge into master.

**Pull request merged**